mgmt/rest/rest_func_test.go

```diff
@@ -582,6 +583,12 @@ func TestPluginRestCalls(t *testing.T) {
 				cfg := getDefaultMockConfig()
 				err := cfgfile.Read("../../examples/configs/snap-config-sample.json", &cfg, MOCK_CONSTRAINTS)
 				So(err, ShouldBeNil)
+				if len(SNAP_AUTODISCOVER_PATH) == 0 {
+					if len(SNAP_PATH) != 0 {
+						SNAP_AUTODISCOVER_PATH = fmt.Sprintf("%s/plugin", SNAP_PATH)
+					}
+				}
+				cfg.Control.AutoDiscoverPath = SNAP_AUTODISCOVER_PATH
```

Comment:
In fact, we are building plugins with some modifications (basically to support proper running in Docker containers) which I forked in my personal github repository and we'd like to ensure that whenever we modify snap these plugins still load therefore the need of this variable
Is this a bit clearer ?

Do the other parts of this PR sounds OK to you ?

Thanks for everything